SPECS/golang/golang-1.15.spec

```diff
@@ -32,18 +32,18 @@ Go is an open source programming language that makes it easy to build simple, re
 %prep
 # Setup go 1.4 bootstrap source
 tar xf %{SOURCE1} --no-same-owner
-patch -Np1 --ignore-whitespace < /usr/src/mariner/SOURCES/go14_bootstrap_aarch64.patch
+patch -Np1 --ignore-whitespace < %{_prefix}/src/mariner/SOURCES/go14_bootstrap_aarch64.patch
```

Comment:
Can we try the macro for `Patch0` instead (I'm guessing `%{PATCH0}` similar to `%{SOURCE1}` above for instance)? It's shorter and safer, since the file name will follow whatever we do with `Patch0`.